omod/src/main/java/org/openmrs/module/financials/fragment/controller/PharmacyRevenueSummariesFragmentController.java

```diff
@@ -10,33 +11,61 @@
 import org.openmrs.ui.framework.page.PageModel;
 import org.springframework.web.bind.annotation.RequestParam;
 
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
 import java.util.List;
 
 public class PharmacyRevenueSummariesFragmentController {
 	
 	public void controller(PageModel model) {
+		List<PharmacyBillSummary> pharmacyTransactions = new ArrayList<PharmacyBillSummary>();
+		
+		Calendar cal = Calendar.getInstance();
+		cal.add(Calendar.MONTH, -1);
+		SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss");
+		String startAt = formatter.format(cal.getTime());
```

Comment:
for proper coverage, this date should be date part set to cal.gettime and time part be set to 00:00:00